### ISSUE TITLE
Chat background on AMOED Black are now off black

### ISSUE THREAD

**Issue by Sn0whax** [johanw666/Signal-Android#3]
When using 4.47.5.0-JW it appears the background is inherited from dark mode instead of the OLED black theme. The contact list appears to be fine.

Would it also be possible to change the group chat header from blue to OLED black when using group chats?

Thanks again!

**Comment by johanw666** [comment]
I can't remember if it once worked otherwise. It looks black to me but I don't haver an OLED screen so both probably look the same on my device. I'll check the code if I find the time. The header is a separate issue, that is not quickly changed because it would also influence the font color of the text there.

**Comment by Fzndox** [comment]
Yeah, I've tried playing with the group chat header with no luck either.

Regarding the OLED color, it has changed for sure. I tried reverting back to some of your earlier versions where you had started implementing it and the chats were a pure black.
I can also look into it when i have some time.

**Comment by johanw666** [comment]
The theme system was seriously overhauled in 4.50, I've made some changes in 4.50.5.1-JW to correct some theme issues. The amoled theme should be fixed by them, some other themes still need some refinement.

**Comment by Fzndox** [comment]
*I'd like to reopen this*
I've tried 4.50.5.1 and the latest 4.55.8.0-JW. The main list of chats is OLED black. However, once you enter an individual chat the background is the stock, off-black. 
I continue to use 4.38.3.0-JW, there may be slightly later versions that appear to have this fix but I randomly selected a few versions between 4.50.5.1 with it continuing to be off grey.
Is there any way to compare the chats in 4.38.3.0-JW to the latest version and update the grey to the black?

**Comment by johanw666** [comment]
Which version of Android are you using? There are some different settings for different versions because some Android versions don't support all options that are used. I might have overlooked something here.

**Comment by Fzndox** [comment]
I've tested this on a s8 in android 8 and a s10 on android 9. I can also test it next week on a OnePlus 7t on android 10.

I've tried playing with the systemwide dark mode but it doesn't change this app.

**Comment by Fzndox** [comment]
This also happens for the OnePlus 7T on android 10.

**Comment by Fzndox** [comment]
Check out:
https://github.com/mollyim/mollyim-android

They have managed to make it all OLED black, sadly they removed the ability to text. I believe this may resolve the OLED issues.

**Comment by johanw666** [comment]
I removed the extra themes some time ago because maintaining them was too much work with the constant UI redesigns. I am open to new features, to a limit: it has to be maintainable for me, and I must see the need for the function.

Using the Molly code would mean I replace the current dark theme with it, which would be bad for those people prefering  using dark mode as it is now.

**Comment by thrdroom** [comment]
Thanks for fast answer 🙂
 
> Using the Molly code would mean I replace the current dark theme with it, which would be bad for those people prefering using dark mode as it is now.

Could mollys amoled theme and default non amoled dark theme co exist at the same time (selectable in settings)?
It sounds like you don't want to maintain your own themes because of the many changes from vanilla signal.
That way you would "only" need to maintain the theme-selection part in the settings and the rest of the code comes from outside repos. 
So could it be an option to always merge the amoled theme from molly, into your existing code?

Or is this out of the scope too?

**Comment by johanw666** [comment]
> Could mollys amoled theme and default non amoled dark theme co exist at the same time (selectable in settings)?

Yes, that was the solution I had before. However, I kept spending hours to maintin that, and honestly all for a solution I don't even use myself (I use the light theme myself).
